src/amo/pages/Addon/index.js

```diff
@@ -622,10 +622,15 @@ export function mapStateToProps(state, ownProps) {
 
   let addonsByAuthors;
   let installedAddon = {};
+  let currentVersion = null;
 
   if (addon) {
     addonsByAuthors = getAddonsForSlug(state.addonsByAuthors, addon.slug);
     installedAddon = state.installations[addon.guid] || {};
+    currentVersion = getVersionById({
```

Comment:
I don't really like that this selector does not follow the existing conventions (that is: `getVersionId(versionsState,  id)`) but 🤷‍♂️

Reply:
Do you mean because it accepts an object as opposed to two positional arguments?

Reply:
We don't seem to be 100% consistent about that, e.g., https://github.com/mozilla/addons-frontend/blob/2018.10.25/src/amo/reducers/recommendations.js#L118-L126 (although admittedly I wrote that one too). Do feel that's an important convention to maintain? I feel like using an object is more explicit.

Reply:
I always try to avoid positional arguments whenever possible but that's mostly due to my experience with JavaScript before Flow and my experience with Flow before strict arity. Even after all that, positional arguments made up of primitives are still not very type safe -- it's hard for the compiler to detect if they are out of order.

Reply:
> Do you mean because it accepts an object as opposed to two positional arguments?

Yes.

I know we use objects and functions with arity = 1 a lot. I tend to like it because I am a fan of named arguments. That was just a comment I left because I could not simply skip it. I am not sure there is any actionable task behind it, except maybe discussing conventions at the All Hands.